feconf.py

```diff
@@ -381,6 +381,9 @@ def get_empty_ratings():
 # No. of questions to be displayed on a page in the topic editor.
 NUM_QUESTIONS_PER_PAGE = 10
 
+# No. of pretest questions to display.
```

Comment:
This is more like the number to load at once. I don't think we display a fixed number? (But check @vinitamurthi's doc.)

Reply:
Done, added another PretestHandler that manages sending pretests in batches.